client/base.jsx

```diff
@@ -0,0 +1,43 @@
+
+/**
+ * Module dependencies.
+ */
+
+var React = require('react');
+
+module.exports = React.createClass({
+	render: function() {
+
+		var googleMapsScript =
+		    'function initialize() { dispatcher.dispatch(\'GOOGLE_MAPS_READY\'); } ' +
```

Comment:
Why don't we standardize on 2 spaces. That seems to be a standard for javascript these days. However, I think cleaning it up should be a separate single commit...